tastypie_swagger/urls.py

```diff
@@ -1,13 +1,23 @@
 try:
-	from django.conf.urls import patterns, include, url
+    from django.conf.urls import url
 except ImportError:
-	from django.conf.urls.defaults import patterns, include, url
+    from django.conf.urls.defaults import url
 
 from .views import SwaggerView, ResourcesView, SchemaView
 
-urlpatterns = patterns('',
+urlpatterns = [
     url(r'^$', SwaggerView.as_view(), name='index'),
     url(r'^resources/$', ResourcesView.as_view(), name='resources'),
     url(r'^schema/(?P<resource>\S+)$', SchemaView.as_view()),
-    url(r'^schema/$', SchemaView.as_view(), name='schema'),
-)
+    url(r'^schema/$', SchemaView.as_view(), name='schema')
+]
+
+from django import get_version
+from distutils.version import StrictVersion
+
+if StrictVersion(get_version()) < StrictVersion("1.10"):
```

Comment:
May want to change this to check for a version before 1.8 or 1.9 as the new list-style url patterns are supported in both of those versions and 1.9 throws a warning with the wrapped patterns.